test/commands/init.js

```diff
@@ -48,5 +58,7 @@ test('prepare project template', async t => {
   t.falsy(await fs.pathExists(repoPath))
   t.is(undefined, packageJson.license)
   t.falsy(fs.pathExistsSync(licensePath))
-  t.is(appName, project.environments.default.appName)
+  t.is(`${appName}`, project.environments.default.appName)
+  t.is(`${appName}.open.aragonpm.eth`, project.environments.staging.appName)
+  t.is(`${appName}.open.aragonpm.eth`, project.environments.production.appName)
```

Comment:
Looks like `appName` for staging and prod will be `aragon-app.aragonpm.eth.open.aragonpm.eth`